crux-bench/src/crux/bench/watdiv_crux.clj

```diff
@@ -7,28 +7,41 @@
             [crux.sparql :as sparql])
   (:import (java.time Duration)))
 
+(defn parse-results [resource]
+  (with-open [rdr (io/reader resource)]
+    (let [rdr-lines (line-seq rdr)]
+      {:ingest (-> (first rdr-lines)
+                   (read-string)
+                   (:time-taken-ms)
+                   (Duration/ofMillis)
+                   (.toString))
+       :queries (->> (rest rdr-lines)
+                     (map read-string)
+                     (filter :result-count)
+                     (map (juxt :query-idx identity))
+                     (into {}))})))
+
+(def parsed-db-results
+  {:rdf4j (some-> (bench/load-from-s3 "rdf4j-3.0.0/rdf4j-20200406-170508Z.edn") parse-results)
+   :neo4j (some-> (bench/load-from-s3 "neo4j-4.0.0/neo4j-20200406-171121Z.edn") parse-results)
+   :datomic (some-> (bench/load-from-s3 "datomic-0.9.5697/datomic-20200406-170841Z.edn") parse-results)})
+
 (defn ingest-crux
   [node]
   (bench/run-bench :ingest
                    (let [{:keys [last-tx entity-count]} (with-open [in (io/input-stream watdiv/watdiv-input-file)]
                                                           (rdf/submit-ntriples node in 1000))]
                      (crux/await-tx node last-tx)
                      {:entity-count entity-count
-                      :node-size-bytes (bench/node-size-in-bytes node)})))
-
-(defn parse-results [resource]
-  (with-open [rdr (io/reader resource)]
-    (->> (line-seq rdr)
-         (map read-string)
-         (filter :result-count)
-         (map (juxt :query-idx identity))
-         (into {}))))
+                      :node-size-bytes (bench/node-size-in-bytes node)
+                      :neo4j-time-taken (get-in parsed-db-results [:neo4j :ingest])
```

Comment:
units, maybe?

Reply:
oh, I see, these are Durations. given we're still in raw-data territory here, (indeed, this is what's getting sent to CloudWatch?) let's try migrate the conversion to strings until we're rendering the results?

Reply:
In which case, should move the conversion to a Duration in general.